src/java.base/share/classes/java/security/doc-files/security-related-system-properties.html

```diff
@@ -0,0 +1,192 @@
+    </tr>
+    <tr>
+        <th scope="row"><code>configparser</code></th>
+        <td>JAAS configuration file parsing</td>
+    </tr>
+    <tr>
+        <th scope="row"><code>gssloginconfig</code></th>
+        <td>Java GSS (Generic Security Services) login configuration file debugging</td>
+    </tr>
+    <tr>
+        <th scope="row"><code>jar</code></th>
+        <td>JAR file verification</td>
+    </tr>
+    <tr>
+        <th scope="row"><code>jca</code></th>
+        <td>JCA engine class debugging</td>
+    </tr>
+    <tr>
+        <th scope="row"><code>KeyStore</code></th>
+        <td><code>KeyStore</code> debugging</td>
+    </tr>
+    <tr>
+        <th scope="row"><code>logincontext</code></th>
+        <td><code>LoginContext</code> results</td>
+    </tr>
+    <tr>
+        <th scope="row"><code>pcsc</code></th>
+        <td>Java Smart Card I/O and SunPCSC provider debugging</td>
+    </tr>
+    <tr>
+        <th scope="row"><code>pkcs11</code></th>
+        <td>PKCS11 session manager debugging</td>
+    </tr>
+    <tr>
+        <th scope="row"><code>pkcs11keystore</code></th>
+        <td>PKCS11 <code>KeyStore</code> debugging</td>
+    </tr>
+    <tr>
+        <th scope="row"><code>pkcs12</code></th>
+        <td>PKCS12 <code>KeyStore</code> debugging</td>
+    </tr>
+    <tr>
+        <th scope="row"><code>properties</code></th>
+        <td><code>java.security</code> configuration file debugging</td>
+    </tr>
+    <tr>
+        <th scope="row"><code>provider</code></th>
+        <td>Security provider debugging. The following engines can be used with the <code>provider</code> sub-option:
+            <code>engine=(engines)</code> : The output is displayed only for a specified list of JCA engines.
+            The supported values for (engines) are:
+            <ul>
+                <li><code>Cipher</code></li>
+                <li><code>KeyAgreement</code></li>
+                <li><code>KeyGenerator</code></li>
+                <li><code>KeyPairGenerator</code></li>
+                <li><code>KeyStore</code></li>
+                <li><code>Mac</code></li>
+                <li><code>MessageDigest</code></li>
+                <li><code>SecureRandom</code></li>
+                <li><code>Signature</code></li>
+            </ul>
+        </td>
+    </tr>
+    <tr>
+        <th scope="row"><code>scl</code></th>
+        <td>Permissions that <code>SecureClassLoader</code> assigns</td>
+    </tr>
+    <tr>
+        <th scope="row"><code>securerandom</code></th>
+        <td><code>SecureRandom</code> debugging</td>
+    </tr>
+    <tr>
+        <th scope="row"><code>sunpkcs11</code></th>
+        <td>SunPKCS11 provider debugging</td>
+    </tr>
+    <tr>
+        <th scope="row"><code>ts</code></th>
+        <td>Timestamping debugging</td>
+    </tr>
+    <tr>
+        <th scope="row"><code>x509</code></th>
+        <td>X.509 certificate debugging. You can use the following sub-option with <code>X.509</code>
```

Comment:
Change the second sentence to "The following sub-option can be used with the <code>X.509</code> option:"